lisa/messages.py

```diff
@@ -232,6 +232,13 @@ class NetworkUDPPerformanceMessage(PerfMessage):
     packet_size_kbytes: Decimal = Decimal(0)
 
 
+@dataclass
+class AverageMetric(PerfMessage):
```

Comment:
It should be perf metrics, instead of general names. Like network, there are pps, throughput and so on.

Reply:
I realised that we still need to two different metrics. But I have renamed it to make it specific to what we are testing rather than the tool we use it to test